sktime/classification/base.py

```diff
@@ -150,7 +150,11 @@ def predict_proba(self, X):
         -------
         y : array-like, shape =  [n_instances, n_classes] - predictive pmf
         """
-        raise NotImplementedError("abstract method")
+        coerce_to_numpy = self.get_class_tag("coerce-X-to-numpy", False)
```

Comment:
I think you should be using `get_tag` here - `get_tag` does the same but also allows dynamic overrides.

Reply:
done 83a9faa1

Reply:
predict_proba seems to still use `get_class_tag`. Very minor but a bit strange to use one in fit/predict and another in predict_proba unless theres a reason.